module/platform/Webots/src/Webots.cpp

```diff
@@ -312,8 +312,8 @@ namespace module::platform {
         });
 
         // This trigger updates our current servo state
-        on<Trigger<ServoTargets>, With<RawSensors>>().then([this](const ServoTargets& targets,
-                                                                  const RawSensors& sensors) {
+        on<Trigger<ServoTargets>, With<RawSensors>, Sync<ServoTargets>>().then([this](const ServoTargets& targets,
```

Comment:
We shouldn't sync on `ServoTargets`, we should sync on a type local to the webots module.

Reply:
This was an @Bidski suggestion implemented.. not sure there's anything else you would want to sync on..?

Reply:
This may be a point. We shouldn't sync on `Webots` as this would sync us with non-related reactions. Maybe `ServoState`? If a good type can't be found then a mutex would be a simpler option

Reply:
We can always make an empty struct to sync on.

Reply:
I'll add a mutex to the servostate instance the webots reactor owns then.

Reply:
instead of a `Sync`